.changeset/fifty-maps-drive.md

```diff
@@ -0,0 +1,36 @@
+---
+'@backstage/plugin-scaffolder-react': major
+---
+
+**BREAKING**: Fixed a [wrong behaviour in `SecretWidget`](https://github.com/backstage/backstage/issues/25966) where secrets are saved using the field name instead of the proper path in the formData.
```

Comment:
```suggestion
Fixed a [wrong behaviour in `SecretWidget`](https://github.com/backstage/backstage/issues/25966) where secrets are saved using the field name instead of the proper path in the `formData`. If you were using nested `SecretFields` they will now be on the correct nested value.
```